packages/experiments/src/components/Button/Button.styles.ts

```diff
@@ -164,7 +166,14 @@ export const ButtonStyles: IButtonComponent['styles'] = (props, theme, tokens):
 
   return {
     root: [
-      !circular && getFocusStyle(theme, { inset: 1 }),
+      !circular && getFocusStyle(theme, { inset: 1, outlineColor: tokens.outlineColor }),
+      circular && {
+        selectors: {
+          [`.${IsFocusVisibleClassName} &:focus`]: {
```

Comment:
We should use `getFocusStyle` helper wherever we can. Seems like it could work here

Reply:
We cannot just use `getFocusStyle` for circular Buttons because this is how they end up looking:

![image](https://user-images.githubusercontent.com/7798177/57258442-4b348980-7011-11e9-8e4a-75da4aad66c2.png)